### DISCUSSION TITLE
Conbee/ (Deconz/Zigbee2mqtt) - Mancata rilevazione sensori

### DISCUSSION THREAD

**Discussion by tetionet** [indomus/forum#1402] [ZigBee, Z-Wave, Thread, Matter]
Salve,
ho notato circa un mese fa che i sensori zigbee posti a pochi metri di distanza dal conbee non vendono più rilevati. Ho commesso il terribile errore di non eseguire un backup dei dati di deconz col risultato che oggi non riesco a rilevare più nessun sensore.

HW Rpi4B. Conbee installato con cavo di estensione e con Firmware aggiornato. SDD collegata tramite usb 3.0 allontanata tramite cavo usb a doppia schermatura a circa 1,5m di distanza e posto un ripiano al di sotto della scrivania dove attualmente è poggiato l'Rpi. Batteria sostituita ieri (ultima prova eseguita) su uno aqara door/window sensor.

Ambiente HASSOS con add-on Deconz e Zigbee2mqtt correttamente installati e configurati secondo le guide indomus.

Entrambi gli add-on vengono avviati correttamente; Zigbee2mqtt non ha un log particolarmente verboso ma la chiavetta viene riconosciuta senza warning di alcun tipo; sulla pagina web, a valle del "permit to join" non riesco a vedere nessun nuovo sensore, nonostante provi a resettare i sensori o simularne il funionamento.
Deconz ha un log incredibilmente verboso: confermo che la chiavetta viene riconosciuta e il firmware confermato come corretto e ultimo aggiornato. Purtroppo non riesco a valutare il log (non riporta errori nè warning, solo informazioni).

Ho provato ovviamente anche a disinstallare e reinstallare più volte.
Prima di riportare pagine e pagine di log perchè non saprei nemmeno cosa cercare, vorrei sapere cosa suggerite di fare.
Grazie in anticipo.

**Comment by snakuzzo** [comment]
Una cosa è certa...non puoi passare da deconz a zigbee2mqtt senza dover rifare il pairing.
Su Decnoz non so dirti. Con Zigbee2mqtt prova ad impostare il canale 11 e a rimettere in pairing mode i device (abilitando prima il permit join)

**Reply by tetionet** [reply]
Si, era sottinteso. Parto comunque dal presupposto che poichè nessun sensore viene visto ho spento deconz e fatto partire zigbee2mqtt ricominciando da zero la procedura. In entrambi i casi, ogni volta eseguo il permit join e poi resetto il sensore (per deconz avvio la procedura di rilevazione tramite phoscon webapp e poi resetto il sensore).
Il canale è impostato a 11 di default.

**Reply by snakuzzo** [reply]
Mi sembra strano che tu abbia lo stesso problema per tutti i device.
Che ti po di sensori hai ?

**Reply by tetionet** [reply]
4 aqara door/window sensor, 1 aqara flood sensor.

**Reply by snakuzzo** [reply]
Mai fallito un pairing...mi sembra troppo strano.
Giusto per sicurezza (immagino sia scontato) ...per fare i pairing:

- Abiliti permit join su Z2M
- Tieni premuto il pulsantino laterale sul sensore per 6-7s e poi rilasci

Giusto ?

**Reply by tetionet** [reply]
Si, visto che ci sono 2 permit join qualche dubbio me lo hai fatto venire. Comunque il primo nei settings è spuntato, il secondo è un pulsante temporizzato che premo quando devo fare il pairing. 
![image](https://user-images.githubusercontent.com/75281515/151374177-9605cc49-afa1-47a4-8ff3-c00423578f1a.png)
![image](https://user-images.githubusercontent.com/75281515/151374238-60a6b619-1efb-4043-bc40-5de95f49be8c.png)

Il pulsante laterale sul sensore door/window è sufficiente che sia premuto per 3 secondi... comunque ho provato 6 e anche 10. Stesso identico discorso anche per il water sensor.
Il sensore lampeggia velocemente 3 volte, poi pausa, poi lampeggia una sola volta e poi non lampeggia più. Non ricordo se il sensore dovesse lampeggiare all'apertura/chiusura del contatto: se fosse così significa che ho tutte le batterie scariche e anche quella che ho comprato è scarica uguale O_O

**Reply by snakuzzo** [reply]
ma guarda che sono due cose diverse! Non ci sono 2 permit join :D

questo è un flag che se abilitato, avvia z2m con il permit join sempre attivo
![1](https://user-images.githubusercontent.com/7205371/151378175-02bd849c-8026-45a5-893e-3cad2238c5aa.png)

questo invece è il vero tasto permit join che devi premere quando devi collegare un device (e ti parte un timer di circa 260s)
![1](https://user-images.githubusercontent.com/7205371/151378392-66763689-7b02-476b-ac75-546127f7a7b2.png)

**Reply by tetionet** [reply]
Si comunque confermo che il flag è settato, e comunque premo anche il pulsante di permit join per far partire la procedura. Altre idee?

**Reply by snakuzzo** [reply]
Oltra a cambiare canale ed eventualmente pan e network id non saprei che dirti.
Tu sei sempre si curo che clicchi il pulsante correttamente ? Io solitamente premo e tengo premuto per circa 7s...mai fallito un pairing di quei device

**Reply by tetionet** [reply]
Grazie, risolto cambiando lo zigbee channel a 12. Tutto funziona come si deve.

Adesso, mi chiedo, da cosa può dipendere la necessità di spostamento del canale dallo canonico 11 al 12? Nei dintorni non ho altre abitazioni domotizzate, sul Rpi4 ho solo Hassos, come sensori senza fili ho solo shelly e questi 4 sensori..  di "non-standard" ho soltanto il bluetooth attivato per la localizzazione.

**Reply by snakuzzo** [reply]
Interferenze...ma non è che noi possiamo dirti quale canale a casa tua sia meglio dell'altro :D
Queste sono cose che, se non hai apparecchiature professionali, puoi farle andare avanti solo a tentativi...finché non trovi la situazione stabile.
Leggi [QUI](https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html#reduce-wi-fi-interference-by-changing-the-zigbee-channel)

**Reply by KC-inDomus** [reply]
nonché [qui](https://indomus.it/focus/canali-wi-fi-scegliere-al-meglio-per-le-massime-prestazioni/) sul tema canali wi-fi (tema ovviamente... di confine)
